subprojects/gst-plugins-good/ext/qt/gstqsgtexture.cc

```diff
@@ -27,7 +27,7 @@
 
 #include <gst/video/video.h>
 #include <gst/gl/gl.h>
-#include <gst/gl/gstglfuncs.h>
+#include <ext/qt/gstqtgl.h>
```

Comment:
This change is unrelated to the PR topic. Please remove and submit separately.

Reply:
Modified as per the suggestion